clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerInterceptor.java

```diff
@@ -32,6 +32,8 @@
  * just log the errors.
  * <p>
  * ConsumerInterceptor callbacks are called from the same thread that invokes {@link org.apache.kafka.clients.consumer.KafkaConsumer#poll(long)}.
+ * <p>
+ * If you need access to cluster metadata, you will need to implement {@link org.apache.kafka.common.ClusterResourceListener} interface.
```

Comment:
Nitpick: this would read a bit better like (same for other classes that have the same blurb):

"Implement {@link org.apache.kafka.common.ClusterResourceListener} to receive cluster metadata once it's available. Please see the class documentation for ClusterResourceListener for more information."